context/.storybook/main.js

```diff
@@ -1,26 +1,28 @@
-const path = require("path")
-const {alias} = require('../build-utils/alias');
-
+const path = require("path");
+const {
+  alias
+} = require('../build-utils/alias');
 module.exports = {
-  "stories": [
-    '../app/static/js/shared-styles/**/*.stories.js',
-    '../app/static/js/components/**/*.stories.js'
-  ],
-  "addons": [
-    "@storybook/addon-links",
-    "@storybook/addon-essentials"
-  ],
-  "webpackFinal": async (config) => {
+  "stories": ['../app/static/js/shared-styles/**/*.stories.js', '../app/static/js/components/**/*.stories.js'],
+  "addons": ["@storybook/addon-links", "@storybook/addon-essentials"],
+  "webpackFinal": async config => {
```

Comment:
Like we discussed on the call, seems like we're getting a lot of formatting changes here and elsewhere throughout the PR. Do you know what's causing this?

Reply:
To my knowledge, these changes were automatically performed by Storybook's script, while the automatic changes in this commit: https://github.com/hubmapconsortium/portal-ui/pull/3121/commits/e4a74f8589bd60309053c5ebfacf915336ee9cff were performed because they started throwing errors after the webpack 5 update. I haven't set up anything new in terms of lint config, so any applied rules should be coming from the existing `.eslintrc.yml` (and by extension, `.prettierrc.yml`).